### ISSUE TITLE
File - Import V1 File errors

### ISSUE THREAD

**Issue by DougSK** [Maetrim/DDOBuilderV2#201]
I'm pulling all my individual DDOBuilder V1 files for my main into one central V2 file with all its lives combined.

And it's working quite well.  Just a couple of glitchy things:

1. It doesn't seem to verify the validity of the build when it imports it.  Evidence in the following.

2.  On two Warlock builds, it set the Alignment to Lawful Good.  Both were Fiends.  That's not a valid Alignment for a Fiend Warlock.  It didn't flag the Pact or Pact spells as wrong until I changed the Build Level setting.  Then it appears to have run a verification routine and dumped all the Fiend-specific Feats (and spells, I think, though I didn't explicitly check that).  I reopened the offending life files in V1 and the alignments were correctly set there.  No idea why the V2 Import function changed them.  They were set to True Neutral.  Another build (Druid) with a Lawful Neutral alignment carried the old alignment into V2 without issues.
In addition, when opening the build in V1, it verified the build and updated some kind of older version of those Pact Feats by dumping and then replacing them.  Idr if it did something similar in V2 when I changed the Build Level setting or not.

3. In a similar vein, some of these older builds used the Completionist Feat.  When they took it, it was valid.  Since then, additions to the game have invalidated it for them.  Re-opening the V1 Build in the V1 planner verifies that those Feats are invalid and removes them.  Importing the V1 Build into the V2 planner does not.

**Comment by DougSK** [comment]
Apologies for the misdirect on the Completionist Feat.  It's now an autogranted Feat, which is why the V1 planner removes it when I open an older build file.  The V2 planner doesn't do that until I change something else (I trigger it deliberately by changing the Build Level.

**Comment by Maetrim** [comment]
1. Build verification on load is something I have yet to implement fully. Will come at som epoint.

2. I found an issue with Alignment import for "True Neutral" from V1 which was called "Neutral" in V1 and "True Neutral" in V2 so does not translate correctly. A patch for legacy import was added to a lookup table.

3. Feats such as this will be revoked at the first action that cause an internal character validation event. It works, but its not pretty. Will hopefully be fixed when I finally action item 1 above,.
